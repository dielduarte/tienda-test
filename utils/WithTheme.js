import React from 'react';
import { ThemeProvider, injectGlobal } from 'styled-components';

injectGlobal`
  * { 
    margin: 0;
    padding: 0;
    box-sizing: border-box;
    font-family: 'Source Sans Pro', Helvetica,sans-serif;
  }
`;

const theme = {
  white: '#FFF',
  darkGray: '#333',
  gray: '#b5b5b5',
  lightGray: '#f5f5f5',
  lightBlue: '#68bcce',
  red: 'red'
};

function WithTheme({ children }) {
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
}

export default WithTheme;
