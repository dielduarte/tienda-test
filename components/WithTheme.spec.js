import { shallow } from 'enzyme';
import 'jest-styled-components';

import WithTheme from './WithTheme';

describe('WithTheme', () => {
  it('Regular', () => {
    const wrapper = shallow(<WithTheme />);
    expect(wrapper).toMatchSnapshot();
  });
});
