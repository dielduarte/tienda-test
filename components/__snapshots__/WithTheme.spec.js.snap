// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithTheme Regular 1`] = `
<ThemeProvider
  theme={
    Object {
      "darkGray": "#333",
      "gray": "#b5b5b5",
      "lightBlue": "#68bcce",
      "lightGray": "#f5f5f5",
      "white": "#FFF",
    }
  }
/>
`;
